File: BackEnd/TelegramBot.Application/Services/ChannelService/DTOModel/ChannelInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using AutoMapper;
using TelegramBot.Core.Entities;

namespace TelegramBot.Application.Services.ChannelService.DTOModel
{
    public class ChannelInfoModel
    {
        public string? Name { get; set; }
        public string? Username { get; set; }
        public string? Description { get; set; }
        public string? Phone { get; set; }
        public long ChannelTelegramId { get; set; }

        private class Mapping : Profile{
            public Mapping(){
                CreateMap<ChannelInfoModel , ChannelInfoEntity>().ReverseMap();
            }
        }
    }
}

File: Bot/Models/ChannelInfoModel.py
from typing import Optional

class ChannelInfoModel:
    def __init__(self, name: Optional[str], username: Optional[str], description: Optional[str], phone: Optional[str], channelTelegramId: int):
        self.name = name
        self.Username = username
        self.description = description
        self.phone = phone
        self.channelTelegramId = channelTelegramId

    def to_dict(self):
        return {
            "name": self.name,
            "Username": self.Username,
            "description": self.description,
            "phone": self.phone,
            "channelTelegramId": self.channelTelegramId
        }
